public/locales/en/status.json

```diff
@@ -34,7 +34,7 @@
     "error": {
       "required": "The given name(s) is required."
     },
-    "help-message": "Enter first name(s) <strong>and any middle name(s)</strong> of the person who the passport is for.",
+    "help-message": "Enter first name(s) <strong>and any middle name(s)</strong> of the person who the passport is for. <strong>Make sure the information you enter matches your passport application.</strong>",
```

Comment:
We would also need to add these details on the 'email' page

Reply:
Donezo.